src/components/Quote.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import spinLogo from './arrow-clockwise.svg';
+
+const Quote = () => {
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(null);
+  const apiKey = 'vA0/JB/b2F3uIe+n+OUoAA==ssQu65YPz7YnNSOQ';
+
+  const types = ['communications', 'computers', 'cool', 'courage', 'education', 'experience', 'faith', 'family',
+    'famous', 'leadership', 'learning', 'money', 'morning', 'movies', 'success', 'legal', 'life', 'love',
+  ];
+  const category = types[Math.floor(10 * Math.random(types.length - 2))];
+  const fetchQuoteData = async () => {
+    try {
+      const response = await fetch(`https://api.api-ninjas.com/v1/quotes?category=${category}`, {
+        headers: {
+          'X-Api-Key': apiKey,
+        },
+      });
+      const json = await response.json();
+      setData(json);
+      console.log('JSON response: ', json);
+      console.log('data: ', data);
```

Comment:
- [ ] @AliRezaBmeDu Great work on your Quote component however it's not good practice to have console.log statements in your codebase, kindly consider removing them.